showcase/app/templates/components/stepper/nav.hbs

```diff
@@ -0,0 +1,464 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Stepper Component"}}
+
+<Shw::Text::H1>Stepper::Nav</Shw::Text::H1>
+
+<section data-test-percy>
+
+  <Shw::Text::H2>Interactivity</Shw::Text::H2>
+
+  <Shw::Text::Body>Interactive</Shw::Text::Body>
+
```

Comment:
I would use `<Shw::Grid @columns={{1}} as |SG|>` with SG.Item labels for these. Then you can add whatever spacing seems good between them easily and use a consistent text style for label text such as "Interactive" & "Non-interactive"

Reply:
Also reformat the examples below to use `Shw::Grid` etc.